Library/Formula/jdnssec-tools.rb

```diff
@@ -0,0 +1,21 @@
+class JdnssecTools < Formula
+  desc "Java command-line tools for DNSSEC"
+  homepage "http://www.verisignlabs.com/jdnssec-tools/"
+  url "http://www.verisignlabs.com/dnssec-tools/packages/jdnssec-tools-0.12.tar.gz"
+  sha256 "3fa0bfe062051d9048c5186adbb04d8f5e176e50da3cdcf9b0d97e1c558efc22"
+
+  depends_on :java
+
+  def install
+    inreplace Dir["bin/*"] do |s|
+      s.gsub! /basedir=.*/, "basedir=#{libexec}"
+    end
+    bin.install Dir["bin/*"]
```

Comment:
The scripts I install now (as provided by upstream) put all three .jar files in `CLASSPATH`; I don't see (mostly because I'm not very experienced with Java) how to make the calls work without including the same `CLASSPATH` either in env or in `-cp`. Looking at other formulas (like gwt) I see not using `write_jar_script` is not uncommon for these cases.

For reference, jdnssec-dstool looks like this (and the others basically differ by one word, although some don't have the ulimit block):
```bash
#! /bin/sh

thisdir=`dirname $0`
basedir=/usr/local/Cellar/jdnssec-tools/0.12/libexec

ulimit_max=`ulimit -H -n`
if [ $ulimit_max != "unlimited" ]; then
    ulimit -n $ulimit_max
fi

# set the classpath
for i in $basedir/lib/*.jar $basedir/lib/*.zip $basedir/build/lib/*.jar; do
  CLASSPATH="$CLASSPATH":"$i"
done
export CLASSPATH

exec java com.verisignlabs.dnssec.cl.DSTool "$@"
```